pyartcd/pyartcd/pipelines/ocp4.py

```diff
@@ -220,6 +230,9 @@ def _plan_pinned_builds(self):
         if not self.build_plan.build_images:
             run_details.update_description('Images: not building.<br/>')
 
+        elif self.mass_rebuild:
+            run_details.update_description('Mass image rebuild (more than half) - invoking serializing semaphore')
```

Comment:
Would like to update the title as well to indicate this is a mass rebuild.

Reply:
Done in https://github.com/openshift-eng/aos-cd-jobs/pull/3791/commits/a93433f3e26d2c42cf823ee80397362bc915cb4e